std/types/coin_signature.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
+from blspy import G1Element
+import chia.wallet.sign_coin_solutions as scs
```

Comment:
There is no package `chia` in this project I'm pretty sure.  You could grab the `sign_coin_solutions` file from `chia_blockchain` and throw it into the std.util folder though.